### ISSUE TITLE
Accessibility note for animating text components

### ISSUE THREAD

**Issue by emileenoh** [mciastek/sal#82]
I was testing a site I'm developing that uses sal.js, and I discovered that text elements with the `fade-in` animation are not read by a screen reader and cannot be focused on which is a WCAG violation.

Is there any way to configure the starting opacity of elements that use fade-in to start at opacity 1 instead of opacity 0?

**Comment by mciastek** [comment]
Regarding a11y I can recommend adding [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) attribute. I think it could be difficult to make sal.js fully a11y compatible.

As for different opacity value, you can define your own styles for animation see this comment - https://github.com/mciastek/sal/issues/17#issuecomment-480833686.
